src/index.js

```diff
@@ -4,7 +4,7 @@ const messageHandler = require('./handlers/message.handler');
 const guildCreateHandler = require('./handlers/guild-create.handler');
 const logger = require('./util/logger');
 
-const client = new Discord.Client();
+const client = new Discord.Client({ intents: new Discord.Intents(37379) });
```

Comment:
I'm unfamiliar with this. Can you tell me what it does and what the `37379` constant means?

Reply:
Basically its the Bitfield for the Intents, which discords websocket uses to know which Events to send you. You can calculate them or like use websites like this https://ziad87.net/intents/. 37379 Is basically the intent bitfield for GUILDS, GUILD_MEMBERS, GUILD_MESSAGES, DIRECT_MESSAGES and MESSAGE_CONTENT. Should pretty much give you all the events you would need.

Reply:
Nice.

As we discussed offline, we're going to change this code to use the array syntax with constants (like [this](https://discordjs.guide/popular-topics/intents.html#enabling-intents) so that the list of intents is clear to someone reading the code

And we're also going to reduce the list of intents to the minimum habla needs which is GUILDS, GUILDS_MESSAGES and MESSAGE_CONTENT